app/views/IleCodeMapper.scala

```diff
@@ -15,31 +15,32 @@
  */
 
 package views
+
+import javax.inject.{Inject, Singleton}
+import models.viewmodels.decoder.{CodeWithMessageKey, Decoder, ROECode}
 import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{Content, Empty, HtmlContent, Text}
 
-object IleCodeMapper {
+@Singleton
+class IleCodeMapper @Inject()(decoder: Decoder) {
```

Comment:
👍 
Nit: Maybe a name that describes more specifically it's use case? Because this is returning a content specific to "Consignment Details" page